app/experimenter/experiments/tests/factories/nimbus.py

```diff
@@ -45,11 +45,12 @@ class Meta:
         model = NimbusExperiment
 
     @classmethod
-    def create_with_status(cls, target_status, **kwargs):
+    def create_with_status(cls, target_status=NimbusExperiment.Status.DRAFT, **kwargs):
```

Comment:
I wouldn't use a default argument here, we've always tried to be explicit about the status a factory experiment is created with right in the test for readability.